engine/src/witness/eth.rs

```diff
@@ -132,8 +132,8 @@ where
 	eth_safe_vault_source
 		.clone()
 		.key_manager_witnessing(process_call.clone(), eth_client.clone(), key_manager_address)
-		.continuous("KeyManager".to_string(), db.clone())
-		.logging("KeyManager")
+		.continuous("EthereumKeyManager".to_string(), db.clone())
```

Comment:
@kylezs isn't this wrong, these ones sohludn't be changed (i.e. left as "KeyManager")?

Reply:
Yes